### PR TITLE
Bring back AdditionalUserDataFiles

### PR DESCRIPTION
**Pull request by ncdc**
Signed-off-by: Andy Goldstein <goldsteina@vmware.com>

**What this PR does / why we need it**:
Bring back what was accidentally removed in #42.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ncdc** [comment]
@amy this will fix the manifest verification failure in your PR

File: api/v1alpha2/kubeadmbootstrapconfig_types.go [hunk 1]
```diff
@@ -33,6 +33,9 @@ type KubeadmConfigSpec struct {
 	// JoinConfiguration is the kubeadm configuration for the join command
 	// +optional
 	JoinConfiguration *kubeadmv1beta1.JoinConfiguration `json:"joinConfiguration,omitempty"`
+	// AdditionalUserDataFiles specifies extra files to be passed to user_data upon creation.
+	// +optional
+	AdditionalUserDataFiles []Files `json:"additionalUserDataFiles,omitempty"`
```
Comment (detiber) [inline api/v1alpha2/kubeadmbootstrapconfig_types.go]: Does this need to be wired up anywhere in the cloudinit code?

Comment (ncdc) [reply api/v1alpha2/kubeadmbootstrapconfig_types.go]: Yes but I'm going to file another issue for that.

**Pull request merged**